### ISSUE TITLE
Support negative values in target variable

### ISSUE THREAD

**Issue by khizanov** [deeplearninc/auger-ml#263]
Since _neg_mean_squared_log_error_ requires positive values for target it throws an exception when this does not hold.
The exception should be thrown **only** if target metric is _neg_mean_squared_log_error_
```
AugerMLTrial.py            267 ERROR    Score neg_mean_squared_log_error for algorithm sklearn.tree.DecisionTreeRegressor failed to build: Mean Squared Logarithmic Error cannot be used when targets contain negative values.
```

**Comment by khizanov** [comment]
This appears only during local test
